### PR TITLE
Feature/mortar 81

### PR DESCRIPTION
**Pull request by xiwcx**

### CODE REVIEW

**Comment by xiwcx** [comment]
- `gulp serve`
- open [http://localhost:3000/typography/]()
- make sure the article body satisfies the requirements

File: app/styles/theme/_bodytext.scss [hunk 1]
```diff
@@ -5,4 +5,33 @@
...
+
+  h2 {
+    @extend %mt_subh1;
+  }
+
+  h3 {
+    @extend %mt_h5;
+  }
+
+  lh {
```
Comment (vamidjo) [inline app/styles/theme/_bodytext.scss]: Is this a thing?

File: app/styles/theme/article.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+          <h3>Header Level 3</h3>
+
+          <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. <em>Aenean ultricies mi vitae est.</em> Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, <code>commodo vitae</code>, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. <a href="#">Donec non enim</a> in turpis pulvinar facilisis. Ut felis.</p>
+
+          <ul>
+            <lh>List Heading</lh>
```
Comment (vamidjo) [inline app/styles/theme/article.html]: lh bugger

File: app/styles/theme/lists.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<div class="pagewrapper mt_row mt_col-gut-lg ">
+  <h3 class="mt_h3">Lists</h3>
+
+  <div class="typographyexamples-item mt_row-gut-3 mt_bgcolor--neutral--xxl">
+    <ul class="mt_list">
+      <lh class="mt_list--header">List Heading</lh>
```
Comment (vamidjo) [inline app/styles/theme/lists.html]: another one

**Pull request merged**